spring_const_graph.py

```diff
@@ -90,14 +87,3 @@ def plot_mass_vs_extension_with_trendline(masses, elongations):
 
     # Show the plot
     plt.show()
```

Comment:
This function does not have a return value so you rely on plot.show to work, see above

Reply:
> This function does not have a return value so you rely on plot.show to work, see above

This function only shows the plot we don't need any output values for it.

Reply:
If the file is run in Jupyter Notebook, it plots without error but I also have fixed the code by adding a save image which is in case when runs in the terminal, it will save the image of the best-fit line too.

Reply:
> If the file is run in Jupyter Notebook, it plots without error but I also have fixed the code by adding a save image which is in case when runs in the terminal, it will save the image of the best-fit line too.